# Final Practice Problems

## Problem 1

In [52]:
xbar = 82.17
sigma = 2.2
mu = 81
alpha = 0.05

z = (xbar - mu) / (sigma / sqrt(12))
paste0("Z-score: ", round(z, 4))
paste0("pnorm(z): ", round(pnorm(z), 4))


z_critical = qnorm(1 - alpha/2)
paste0("Critical Z-value: ", round(z_critical, 4))

if (z > z_critical) {
  result = "Reject the null hypothesis"
} else {
  result = "Fail to reject the null hypothesis"
}
result


[1] "Z-score: 1.8423"

[1] "pnorm(z): 0.9673"

[1] "Critical Z-value: 1.96"

[1] "Fail to reject the null hypothesis"

## Problem 3

In [ ]:
# Observed counts
obs = c(292, 190, 18)

# Hypothesized probabilities: 60%, 35%, 5%
p0 = c(0.60, 0.35, 0.05)

# Chi-square goodness-of-fit test
gof_test = chisq.test(x = obs, p = p0)

gof_test$statistic
gof_test$p.value 

X-squared 
 3.459048

[1] 0.1773689

In [31]:
n  = 500
exp = n * p0

chi_sq = sum((obs - exp)^2 / exp)
p_val  = 1 - pchisq(chi_sq, df = length(obs) - 1)

chi_sq
p_val


[1] 3.459048

[1] 0.1773689

## Problem 4

In [33]:
n = 15
xbar = 3.87
s = 1.45
mu0 = 4.69
alpha = 0.10

t_stat = (xbar - mu0) / (s / sqrt(n))
crit = qt(alpha, df = n - 1)

paste0("t statistic: ", round(t_stat, 4))
paste0("critical value: ", round(crit, 4))

if (t_stat < crit) {
  result = "Reject the null hypothesis"
} else {
  result = "Fail to reject the null hypothesis"
}
result

[1] "t statistic: -2.1902"

[1] "critical value: -1.345"

[1] "Reject the null hypothesis"

## Problem 6: Two-Proportion Test

**Problem Setup:**
We want to test whether there's a significant difference between two population proportions. We have:
- Sample 1: 167 successes out of 200 trials
- Sample 2: 178 successes out of 200 trials

**Hypotheses:**
- $H_0: p_1 = p_2$ (no difference between proportions)
- $H_1: p_1 < p_2$ (proportion 1 is less than proportion 2) - this is a left-tailed test

**Solution Approach:**
1. Calculate sample proportions $\hat{p}_1$ and $\hat{p}_2$
2. Calculate pooled proportion $\hat{p}$ (assuming $H_0$ is true)
3. Calculate standard error using the pooled proportion
4. Compute the test statistic $z = \frac{\hat{p}_1 - \hat{p}_2}{SE}$
5. Find the p-value for the left-tailed test
6. Make a decision based on the significance level

In [34]:
p1 = 167
p2 = 178

p1.hat = p1 / 200
p2.hat = p2 / 200
p.pool = (p1 + p2) / 400
paste0("p pool: ", round(p.pool, 4))

se = sqrt(p.pool * (1 - p.pool) * (1/200 + 1/200))

z = (p1.hat - p2.hat) / se
paste0("Z-score: ", round(z, 4))

p_value = round(pnorm(z), 3)
paste0("P-value: ", p_value)

z_critical = qnorm(0.95)
paste0("Critical Z-value: ", round(z_critical, 4))

if (z > z_critical) {
  result = "Reject the null hypothesis"
} else {
  result = "Fail to reject the null hypothesis"
}
result


[1] "p pool: 0.8625"

[1] "Z-score: -1.5971"

[1] "P-value: 0.055"

[1] "Critical Z-value: 1.6449"

[1] "Fail to reject the null hypothesis"

We test

$$
H_0:p_1=p_2 \quad \text{vs} \quad H_1:p_1<p_2
$$

From the samples:

$$
\hat p_1 = \frac{167}{200} = 0.835,\qquad
\hat p_2 = \frac{178}{200} = 0.89
$$

Pooled proportion under $H_0$:

$$
\hat p = \frac{167+178}{200+200} = \frac{345}{400} = 0.8625
$$

Standard error:

$$
SE = \sqrt{\hat p(1-\hat p)\left(\frac1{n_1}+\frac1{n_2}\right)}
= \sqrt{0.8625(1-0.8625)\left(\frac1{200}+\frac1{200}\right)}
\approx 0.0344
$$

Test statistic:

$$
z = \frac{\hat p_1-\hat p_2}{SE}
= \frac{0.835-0.89}{0.0344}
\approx -1.60
$$

P-value for lower-tailed test:

$$
p = P(Z \le -1.60) \approx 0.055
$$



## Problem 7

In [44]:
# Given
n = 15
df = (n - 1)
s = 1.45       # sample SD
sigma_sq = 2          # hypothesized variance
alpha = 0.05

# (1) Critical value for chi-square (upper tail, since H1: sigma^2 > 2)
chi_crit = qchisq(1 - alpha, df = df)

# (2) Corresponding critical value for S^2
S2_crit = sigma_sq * chi_crit / df

paste0("Critical chi-square value: ", round(chi_crit, 4))
paste0("Critical S^2 value: ", round(S2_crit, 4))

# (3) Test statistic
S2 = s^2
paste0("S^2: ", round(S2, 4))

if (S2 > S2_crit) {
  result = "Reject the null hypothesis"
} else {
  result = "Fail to reject the null hypothesis"
} 

paste0("Decision: ", result)

[1] "Critical chi-square value: 23.6848"

[1] "Critical S^2 value: 3.3835"

[1] "S^2: 2.1025"

[1] "Decision: Fail to reject the null hypothesis"

We use the chi-square test for variance.

* $H_0:\sigma^2 \le 2$ vs $H_1:\sigma^2 > 2$
* $(n=15 \Rightarrow df = n-1 = 14)$
* Critical value: $\chi^2_{0.95,14} \approx 23.68$

Reject $H_0$ if

$$
\chi^2 = \frac{(n-1)S^2}{\sigma_0^2} > 23.68
$$


Equivalently,

$$
S^2 > \frac{\sigma_0^2 \chi^2_{0.95,14}}{n-1}
= \frac{2 \times 23.68}{14}
\approx 3.38
$$

So the correct rejection rule is:

Reject $H_0$ in favor of $H_1$ if $S^2 > 3.38$

Here, $S^2 = 1.45^2 \approx 2.10 < 3.38$, so we **fail to reject $H_0$**.
There is not enough evidence at the 0.05 level to conclude that $\sigma^2 > 2$.


## Problem 8

In [ ]:
round(qf(0.05, df1 = 4, df2 = 2), 3)

[1] 0.144

We’re testing

$$
H_0:\sigma_2^2 = \sigma_1^2 \quad \text{vs} \quad H_1:\sigma_2^2 < \sigma_1^2
$$

with test statistic

$$
T = \frac{S_2^2}{S_1^2}.
$$

Under $H_0$, with $n_1 = 3$ and $n_2 = 5$,

$$
\frac{S_2^2}{S_1^2} \sim F_{(n_2-1,; n_1-1)} = F_{(4,2)}.
$$

Since the alternative is “<”, this is a *left–tailed* test, so the critical value is the 5th percentile of the $F_{4,2}$ distribution:

$$
f_{0.05} = F^{-1}(0.05; 4, 2) \approx 0.144.
$$

So the appropriate critical value (that you would then use to solve for (c)) is

$$
\boxed{0.144}
$$




## Problem 9

In [63]:
alpha = 0.08
tbl = matrix(c(26, 18, 29, 37), nrow = 2, byrow = TRUE)

# Chi-square test of independence
chi_test = chisq.test(tbl, correct = FALSE)
paste0("test statistic: ", round(chi_test$statistic, 3))

# Critical value for alpha = 0.08, df = 1
crit_value = round(qchisq(1-alpha, df=1), 3)
paste0("Critical value: ", crit_value)

if(chi_test$statistic > crit_value) {
  result = "Reject the null hypothesis"
} else {
  result = "Fail to reject the null hypothesis"
}
paste0("Decision: ", result)


[1] "test statistic: 2.424"

[1] "Critical value: 3.065"

[1] "Decision: Fail to reject the null hypothesis"

First set up the contingency table.

$$
\begin{array}{c|cc|c}
& A\text{ Yes} & A\text{ No} & \text{Row total} \\
\hline
B\text{ Yes} & 26 & 18 & 44 \\
B\text{ No}  & 29 & 37 & 66 \\
\hline
\text{Col total} & 55 & 55 & 110
\end{array}
$$

**Expected counts under independence**

$$
E_{ij} = \frac{(\text{row total})(\text{col total})}{110}
$$

So

* $E_{11}=44 \cdot 55/110 = 22$
* $E_{12}=22$
* $E_{21}=33$
* $E_{22}=33$

**Chi-square test statistic**

$$
\chi^2 = \sum \frac{(O-E)^2}{E}
= \frac{(26-22)^2}{22} + \frac{(18-22)^2}{22}
+ \frac{(29-33)^2}{33} + \frac{(37-33)^2}{33}
\approx 2.42
$$

Degrees of freedom:

$$
df = (2-1)(2-1) = 1
$$

**Critical value at $\alpha = 0.08$** (right-tailed test):

$$
\chi^2_{0.92,1} \approx 3.06
$$

**Decision**

$$
\chi^2_{\text{obs}} = 2.42 < 3.06
$$

So we **fail to reject $H_0$** at the 0.08 level. There is not enough evidence to conclude that having trait A is associated with (not independent of) having trait B.